appinventor/appengine/tests/com/google/appinventor/client/youngandroid/TextValidatorsTest.java

```diff
@@ -45,4 +48,13 @@ public void testComponentIdentifierFilter(){
 	      assertFalse(TextValidators.isValidComponentIdentifier(illegalComponentIdentifier));
 	    }
   }
+
+  public void testReservedNameFilter(){
+      for (String reservedWord : reservedWordNames) {
```

Comment:
Indentation for new blocks should be 2 spaces per the [Google Java code style](https://google.github.io/styleguide/javaguide.html).

Reply:
Done. And Intellij defaults fixed to boot.